pkgs/applications/graphics/kcc/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, mkDerivationWith, wrapQtAppsHook, python3Packages, p7zip, p7zipSupport ? true
+}:
+
+mkDerivationWith python3Packages.buildPythonApplication rec {
+  pname = "kcc";
+  version = "5.5.1";
+
+  src = python3Packages.fetchPypi {
+    inherit version;
+    pname = "KindleComicConverter";
+    sha256 = "5dbee5dc5ee06a07316ae5ebaf21ffa1970094dbae5985ad735e2807ef112644";
+  };
+  
+  nativeBuildInputs = [ wrapQtAppsHook ];
+  dontWrapQtApps = true;
+
+  disabled = !(python3Packages.pythonAtLeast "3.3");
+  propagatedBuildInputs = with python3Packages ; [ pillow pyqt5 psutil python-slugify raven ];
```

Formatting it one per line will make the diffs cleaner.

```suggestion
  propagatedBuildInputs = with python3Packages ; [
    pillow
    pyqt5
    psutil
    python-slugify
    raven
  ];
```